ibridges/interactive.py

```diff
@@ -0,0 +1,47 @@
+"""Interactive authentication with iRODS server.
+"""
+
+import json
+import os
+from pathlib import Path
+from typing import Optional, Union
+from getpass import getpass
+
+from ibridges.irodsconnector.session import Session
+
+def interactive_auth(password: Optional[str] = None, irods_env_path: Optional[Union[str, Path]] =
+                 os.path.expanduser("~/.irods/irods_environment.json")) -> Session:
+    """Interactive authentication with iRODS server.
+
+    Stores the password in ~/.irods/.irodsA upon success.
+    """
+    if not os.path.exists(irods_env_path):
+        print(f'File not found: {irods_env_path}')
+        raise FileNotFoundError
+
+    if os.path.exists(os.path.expanduser("~/.irods/.irodsA")):
```

Comment:
Are we sure it's always this path?

Reply:
the icommands and other irods clients only use this path. In the python API we could theoretically store the scrambled password also somewhere else but I would strongly  recommend to not do that to be compatible with other clients. 
In the session.write_pam_password I therefore hardcoded the path for the irodsA.